### PR TITLE
App 2290 migrate icon

### PR DESCRIPTION
**Pull request by mrloureed**
Add icons to Prime core package

image

https://viam.atlassian.net/browse/APP-2290

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -73,5 +73,8 @@
   },
   "svelte": "./dist/index.js",
   "types": "./dist/index.d.ts",
-  "type": "module"
+  "type": "module",
+  "dependencies": {
+    "@mdi/js": "^7.2.96"
```
Comment (micheal-parks) [inline packages/core/package.json:78]: Should be a devDependency

File: packages/core/src/lib/icon/icons.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import {
```
Comment (micheal-parks) [inline packages/core/src/lib/icon/icons.ts:1]: up for switching to a wildcard import here?

```ts
import * as MDI from from '@mdi/js';

export const paths: Record<string, string> = {
  alert: MDI.mdiAlert,
  ...

File: packages/legacy/src/elements/select/select.svelte [hunk 3]
```diff
@@ -271,10 +271,10 @@ $: {
     {#if tooltip}
       <v-tooltip text={tooltip}>
         <v-icon
-          class={cx('text-xs', {
+          class={cx('gextext-xs', {
```
Comment (micheal-parks) [inline packages/legacy/src/elements/select/select.svelte:274]: typo?

**Review by ethanlook — COMMENTED** [review]
Do you mind adding the new story to packages/storybook?

File: packages/core/src/lib/icon.spec.ts
Comment (ethanlook) [inline packages/core/src/lib/icon.spec.ts:1]: Could you please put this in the icon directory?

File: packages/core/src/lib/icon/icon.svelte [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  />
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+import cx from 'classnames';
+import { paths } from './icons';
+
+/**
```
Comment (ethanlook) [inline packages/core/src/lib/icon/icon.svelte:18]: Non-exported variables do not need a docstring unless it makes the code more readable. I think the docstring is unnecessary in the case of Size, hasNameProperty, and sizes, but it's ultimately up to you.

**Review by ethanlook — CHANGES_REQUESTED** [review]
Do you mind adding a new story to package/storybook? We're also migrating the storybook as we go.

**Comment by mrloureed** [comment]
> Do you mind adding a new story to package/storybook? We're also migrating the storybook as we go.

Added . Could def be improved but shows some icons . Can do more when I return from vaca.

**Review by ethanlook — APPROVED** [review]

**Pull request merged**